roles/variables/defaults/main.yml

```diff
@@ -164,7 +164,7 @@ config_base_path: "{{ (archive_config_base_path | regex_replace('\\/$','')) if i
 config_prefix: "/etc"
 
 ### Boolean to have cp-ansible download the Confluent CLI
```

Comment:
Please add in comment that it should be enabled in case of  `secrets protection`

Reply:
added